test/packages/policy_groups/manifest.yml

```diff
@@ -0,0 +1,111 @@
+format_version: 1.0.0
+name: aws
+title: AWS
+version: 0.3.17
+license: basic
+description: AWS Integration
+type: integration
+categories:
+  - aws
+  - cloud
+  - network
+  - security
+release: beta
+conditions:
+  kibana.version: '^7.9.0'
+icons:
+  - src: /img/aws-logo.svg
+    title: logo aws
+    size: 32x32
+    type: image/svg+xml
+screenshots:
+  - src: /img/aws-general-dashboard.png
+    title: General AWS dashboard
+    size: 1702x1063
+    type: image/png
+vars:
+  - name: access_key_id
+    type: text
+    title: Access Key ID
+  - name: secret_access_key
+    type: text
+    title: Secret Access Key
+policy_templates:
```

Comment:
@mtojek is this the array from which Kibana only shows the first element today? Or is that something different?

Reply:
Or maybe @jen-huang can weigh in on my question since she's already watching this PR. 🙂

Reply:
yes, this is what Kibana uses the first element of today